tests/testament/categories.nim

```diff
@@ -63,6 +64,52 @@ proc compileRodFiles(r: var TResults, cat: Category, options: string) =
   test "gtkex1", true
   test "gtkex2"
 
+# --------------------- flags tests -------------------------------------------
+
+proc flagTests(r: var TResults, cat: Category, options: string) =
+  template test(filename: untyped, clearCacheFirst=false) =
+    if clearCacheFirst: delNimCache(filename, options)
+    testSpec r, makeTest(flagsDir / filename, options, cat)
+
+  # --genscript
+  test "tgenscript", true
+  test "tgenscript"
+
+  var build = ""
+  var run = ""
+  var cmd = "cd " & flagsDir / "nimcache" & " && "
+  var script = "compile_tgenscript"
+  when defined(windows):
+    cmd = "cmd /c " & cmd & "$#"
+    script &= ".bat"
+
+  when defined(linux):
+    cmd = "bash -c \"" & cmd & "./$#\""
```

Comment:
And if bash not installed?

Reply:
Bash is available on Travis so test passes. Do we need to change this to "sh -c" instead? Do we run full tests on other systems that don't have bash?

Reply:
I've changed it to sh, makes sense.